app/screens/settings/notification_mention/notification_mention.tsx

```diff
@@ -0,0 +1,286 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useReducer, useState} from 'react';
+import {Alert, ScrollView, Text, View} from 'react-native';
+import {SafeAreaView} from 'react-native-safe-area-context';
+
+import FormattedText from '@components/formatted_text';
+import Section from '@components/section';
+import SectionItem from '@components/section_item';
+import {useTheme} from '@context/theme';
+import {t} from '@i18n';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+import type UserModel from '@typings/database/models/servers/user';
+
+const UPDATE_MENTION_PREF = 'UPDATE_MENTION_PREF';
+const INITIAL_STATE = {
+    firstName: false,
+    usernameMention: false,
+    channel: false,
+};
```

Comment:
Possibly move out constants to the `/app/constants`? 0/5 since they're only used here for now.

Reply:
Yes, since they will only be used here, I'm going to house it in this directory